frontend/public/style/_common.scss

```diff
@@ -1,5 +1,10 @@
 $co-external-link-padding-right: 15px;
 
+.co-divider--with-margins {
```

Comment:
Adding a modifier without a corresponding block rule seems strange.  Why not just `.co-divider`?